persistence-api/src/main/java/io/stargate/db/query/builder/QueryBuilderImpl.java

```diff
@@ -85,7 +85,7 @@
       @SubExpr(
           name = "select",
           definedAs =
-              "select star? column* writeTimeColumn? from (where* limit? orderBy*) allowFiltering?"),
+              "select star? column* writeTimeColumn? from (where* perPartitionLimit? limit? orderBy*) allowFiltering?"),
```

Comment:
It's legal to have both a per partition and "overall" limit, right?

Reply:
yes. `PersisteceTest` has a test case for that too.